phase-0-tracks/ruby/vampires.rb

```diff
@@ -0,0 +1,110 @@
+puts "How many employees will be processed?"
+
+employees = gets.to_i
+
+i = 0
+
+until i > employees.to_i
+
+puts "What is your Name?"
+
+e_name = gets.chomp
+
+valid_input = FALSE
+until valid_input 
+
+puts "How old are you? What year were you born?"
+
+age = gets.to_i
+year = gets.to_i
+
+if age.to_i != (2017-year)

```

Comment:
Check out the Ruby Time class and see what you can do with `Time.now.year`.  This will help the code run a but more dynamically.  Otherwise you will have to keep changing this file every new year to keep the code running properly.